portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,45 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/* 
+ * Fade in items on scroll when their upper bound crosses the bottom of 
+ * of the screen.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+$(window).on("load", function() {
+    $(window).scroll(function() {
+        var window_bottom = $(this).scrollTop() + $(this).innerHeight();
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+        $(".fade").each(function() {
+            var obj_bottom = $(this).offset().top + $(this).outerHeight();
+            if (obj_bottom < window_bottom) {
+                if ($(this).css("opacity")==0) {
+                    $(this).fadeTo(500, 1);
+                }
+            } else {
+                if ($(this).css("opacity") == 1) {
+                    $(this).fadeTo(500, 0);
+                }
+            }
+        });
+    }).scroll();
+});
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+/*
+ * Retrieve comments from database. Set limit on the number of comments
+ * retrieved through query parameter.
+ */
+function getComments(limit) {
```

Comment:
The name `getComments` make it sound that this function will return a result. What it really does is more than that though: it also populates the DOM with the results. It would be good to state explain that.